lib/price-import.coffee

```diff
@@ -83,8 +83,8 @@ class PriceImport extends ProductImport
 
   _resolvePriceReferences: (price) =>
     Promise.all [
-      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.id}\"")
-      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.id}\"")
+      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.groupName}\"")
+      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.key}\"")
```

Comment:
Using ID internally sounds good but I think the external interface should accept the actual identifier names to prevent packages from having to rename like here: https://github.com/commercetools/csv-parser-price/blob/master/src/main.js#L133-L143

Thoughts?

btw. second link is dead